src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinternetbankingpayment.phtml

```diff
@@ -1,59 +1,26 @@
 ?>
 
 <ul id="payment_form_<?php echo $code; ?>" class="banks-list form-list" style="display:none;">
-    <!-- SCB -->
-    <li class="item">
-        <input class="validate-one-required-by-name" id="internet_banking_scb" type="radio" name="payment[bank_type]" value="internet_banking_scb" />
-        <label for="internet_banking_scb">
-            <div class="omise-logo-wrapper scb">
-                <div class="logo-scb"></div>
-            </div>
-            <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$this->__('Siam Commercial Bank');?></span><br/>
-            </div>
-        </label>
-    </li>
 
-    <!-- KTB -->
+    <?php foreach($bankCodes as $bankCode) {  $shortCode = str_replace('internet_banking_', '', $bankCode); ?>
     <li class="item">
-        <input class="validate-one-required-by-name" id="internet_banking_ktb" type="radio" name="payment[bank_type]" value="internet_banking_ktb" />
-        <label for="internet_banking_ktb">
-            <div class="omise-logo-wrapper ktb">
-                <div class="logo-ktb"></div>
+        <input class="validate-one-required-by-name" id="<?=$bankCode;?>" type="radio" name="payment[bank_type]" value="<?=$bankCode;?>" />
+        <label for="<?=$bankCode;?>">
+            <div class="omise-logo-wrapper <?=$shortCode;?>">
+                <div class="logo-<?=$shortCode;?>"></div>
             </div>
             <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$this->__('Krungthai Bank');?></span><br/>
+                <span class="title"><?=Mage::helper('omise_gateway')->internetBankingName($bankCode);?></span><br/>
             </div>
         </label>
     </li>
+    <?php } ?>
 
-    <!-- BAY -->
-    <li class="item">
-        <input class="validate-one-required-by-name" id="internet_banking_bay" type="radio" name="payment[bank_type]" value="internet_banking_bay" />
-        <label for="internet_banking_bay">
-            <div class="omise-logo-wrapper bay">
-                <div class="logo-bay"></div>
-            </div>
-            <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$this->__('Krungsri Bank');?></span><br/>
-            </div>
-        </label>
-    </li>
 
-    <!-- BBL -->
-    <li class="item">
-        <input class="validate-one-required-by-name" id="internet_banking_bbl" type="radio" name="payment[bank_type]" value="internet_banking_bbl" />
-        <label for="internet_banking_bbl">
-            <div class="omise-logo-wrapper bbl">
-                <div class="logo-bbl"></div>
-            </div>
-            <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$this->__('Bangkok Bank');?></span><br/>
-            </div>
-        </label>
-    </li>
     <li>
         <span class="rate secondary-text"><?=$this->__('The bank may charge a small fee for the transaction.');?></span>
```

Comment:
instead of `<?=$this->__('The bank may charge a small fee for the transaction.');?>`
I would do: `<?= $this->__('The bank may charge a small fee for the transaction.'); ?>`